DISCLAIMER.md

```diff
@@ -0,0 +1,41 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Copyrights & Trademarks](#copyrights--trademarks)
+* [Guidelines](#guidelines)
+* [Update of Brands](#update-of-brands)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyrights & Trademarks
+
+The Simple Icons project contains a number of icons with specific requirements around their usage. Where this is the case, we aim to provide access to the proper guidelines through the `guidelines` entry in our JSON file, in our NPM package and through links on our [website].
+
+Some icons are also in the project under specific `licenses`, and we will endeavour to include the relevant information in the JSON file. The lack of a license entry in the JSON does not imply there is no license, simply that we haven't catalogued it yet.
+
+We ask that users of icons in the collection read the guidelines and license agreements fully and display the icons in accordance with them and, where applicable, with the necessary permissions.
+
+If a logo or icon includes a registered trademark or trademark symbol we follow the recommendations outline [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.
+
+~Simple Icons cannot be held responsible for any legal activity raised by a brand over our users' usage of their icon.~
+
+## Guidelines
+
+> NOTE: the addition of guidelines to our project is an ongoing project so the absence of a `guidelines` entry for a particular icon does not imply that guidelines for that icon do not exist.
+
+As guidelines are subject to change and, as we rely largely on our community to help us keep our project updated, the guidelines included in our project for a particular icon may be outdated. Therefore we also ask that users of our project do their own due diligence in researching a brand's guidelines. Additionally, we ask you to inform us of outdated brand guidelines through the [GitHub repository].
```

Comment:
I think this is already pretty good, but I mainly feel there is some duplication between these two section so I tried to make them more focused.

Note that the Table of Contents needs to be updated as well.

```suggestion
## Licenses, Copyrights & Trademarks

> NOTE: the addition of licenses to Simple Icons is an ongoing project. Hence, the absence of a licence data for a particular icon does not imply that the icon is not released under a license.

Simple Icons provides data on the license under which icons are available. We ask users to carefully consider this when using an icon. As licenses are subject to change we also ask our users to regularly check if the license of the icons they use have been changed. If an icon includes a registered trademark (`®`) or trademark symbol (`™`) the recommendations outlined [in the Simple Icons Contribution Guidelines] are followed to decide whether to include the symbol or not.

The license under which an icon is available can be found in the icon's entry in the JSON data file in the NPM and Packagist packages.

As we rely largely on our community to help us keep Simple Icons up-to-date, the license data for an icon may be outdated. If you find that the license data provided is outdated, please [submit an issue] to [the Simple Icons GitHub repository].

~Simple Icons cannot be held responsible for any legal activity raised by a brand over our users' usage of their icon.~

## Brand Guidelines

> NOTE: the addition of guidelines to Simple Icons is an ongoing project. Hence, the absence of guidelines for a particular brand does not imply that the brand has no guidelines.

Simple icons provides a link to a brand's _branding guidelines_ (or similar) if the brand provides one. We ask our users to carefully read these guidelines before using the brand's icon. As guidelines are subject to change we also ask our users to regularly check if the brand guidelines of the icons they use have been updated.

The brand guidelines can be found in the icon's entry in the JSON data file in the NPM and Packagist packages, or through the "Brand Guidelines" links on [the Simple Icons website].

As we rely largely on our community to help us keep Simple Icons up-to-date, the guidelines link for a brand may be outdated. If you find that the guidelines are outdated, please [submit an issue] to [the Simple Icons GitHub repository].
```